config.nims

```diff
@@ -50,6 +50,10 @@ else:
 
 --define:chronicles_line_numbers # useful when debugging=
 switch("define", "chronicles_timestamps=RfcUtcTime")
+switch("define", "chronicles_sinks=textlines[stdout],textlines[nocolors,dynamic],textlines[file,nocolors]")
+switch("define", "chronicles_runtime_filtering=on")
+switch("define", "chronicles_default_output_device=dynamic")
+switch("define", "chronicles_log_level=trace")
```

Comment:
Just wondering whether this will propagate correctly to the mobile builds @alexjba ?

Reply:
`config.nims` gets picked up for all nim-related builds that happen in this folder, so mobile builds in particular should not be special this way - when the build system was set up, `NIMFLAGS` was intended for command-line overrides and other "local" preferences to the build (much like `CFLAGS`) while `config.nims` represents the "permanent" compile-time configuration of the application. 

The biggest difference is that with these flags inside `config.nims`, when you "manually" build things with `nim c` without using the makefile, you'll get a consistent logging experience (ie when you build and run a single unit test file).

Reply:
> command line

Ah, command line should override what's in `config.nims`, so it should be safe that way as well - though for long-term, it would turn into a `when(android)` section in `config.nims` proper.